src/opnsense/service/templates/OPNsense/Unbound/core/dnsbl_module.py

```diff
@@ -1,5 +1,5 @@
 """
```

Comment:
no strong feelings on this end of the scripting ;)

Reply:
I was already wondering whether it would be relevant here. Maybe it's better to omit it to stay in line with the examples as well as signify it's not really meant to be executed directly for it's main purpose.